cmake/gui.cmake

```diff
@@ -18,20 +18,34 @@ if (BUILD_GUI)
 
   find_program(YARN NAMES yarn yarnpkg REQUIRED)
   message(STATUS "Building lokinet-gui with yarn ${YARN}, target ${GUI_YARN_TARGET}")
+  set(wine_env)
+  if(WIN32)
+    set(wine_env WINEDEBUG=-all "WINEPREFIX=${PROJECT_BINARY_DIR}/wineprefix")
+  endif()
+
   add_custom_target(lokinet-gui
     COMMAND ${YARN} install --frozen-lockfile &&
-      WINEDEBUG=-all "WINEPREFIX=${PROJECT_BINARY_DIR}/wineprefix" ${YARN} ${GUI_YARN_EXTRA_OPTS} ${GUI_YARN_TARGET}
+      ${wine_env} ${YARN} ${GUI_YARN_EXTRA_OPTS} ${GUI_YARN_TARGET}
     WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/gui")
 
   if(APPLE)
-    add_custom_target(copy_gui ALL
-      DEPENDS lokinet lokinet-extension lokinet-gui
-      # FIXME: we really shouldn't be building inside the source directory but this is npm...
-      COMMAND ${CMAKE_COMMAND} -E copy_directory
-        ${PROJECT_SOURCE_DIR}/lokinet-gui/release/mac/lokinet-gui.app
-        $<TARGET_BUNDLE_DIR:lokinet>
+    add_custom_target(assemble_gui ALL
```

Comment:
is using `ALL` here correct?

Reply:
Yes: that adds it to the "all" target, which is what gets built if you run `ninja` with no target.